README.md

```diff
@@ -122,31 +133,19 @@ Fetches the open orders in a market. Example response:
 }
 ```
 
-> **Note:** Each order has two "prices" -- `longPrice` and `shortPrice`. This is because the same order exists in the order book for short tokens and for long tokens. If you are trading long tokens, then the `longPrice` is the price to consider, and vice versa. The prices are a number from 0 to `market.numTicks`, which is usually 10000 for Veil markets.
-> 
-> A `longPrice` of 6000 on an order where `longSide` is `buy` means that the creator of this order is willing to buy long shares for 0.6 ETH/share *OR* sell short shares for 0.4 ETH/share.
-> 
-> For more information about this, ask a question on our [Discord](https://discord.gg/RcWDAr9)
-
-### `veil.getOrderFills(market: Market, options?: { page: number })`
+### `veil.getOrderFills(market: Market, tokenType: "long" | "short", options?: { page: number })`
 
-Fetches the order fill history in a market. Example response:
+Fetches the order fill history in a market for tokens of type `tokenType` (LONG or SHORT). Example response:
```

Comment:
How are we doing categorical markets? Would we ever support Augur categorical markets?